.y

```diff
@@ -1 +1 @@
-8
+7
```

Comment:
Shouldn't this be `9`?
```suggestion
9
```

Reply:
Yes, I was hoping to release 1.7 with CSM 1.2, but it looks like CASMCMS-7645 already bumped the minor version (twice).